tests/testutils/collector_process.go

```diff
@@ -43,7 +43,7 @@ type CollectorProcess struct {
 
 // To be used as a builder whose Build() method provides the actual instance capable of launching the process.
```

Comment:
Not part of this diff, but my IDE (goland) flags these comments as warnings because they don't start with the name of the function.

Reply:
Do you know what linter triggers that?  We could add it to the golangci-lint configuration and update as needed in another PR.

Reply:
Apparently staticcheck will do it https://staticcheck.io/docs/checks#ST1020
staticcheck appears to be enabled but maybe needs to be configured to do this check?

Reply:
I experimented with telling staticcheck to do all the checks, and it will report these issues elsewhere, but for some reason not in this directory (maybe because it's named test/testutils idk).